runtime/relevance.js

```diff
@@ -26,8 +26,7 @@ class Relevance {
   relevanceScore() {
```

Comment:
Suggest using a verb-phrase instead of noun-phrase for method that does work. `scoreRelevance` or `calcRelevanceScore', or what have you.